easybuild/easyblocks/i/icc.py

```diff
@@ -89,127 +88,124 @@ def install_step(self):
     def sanity_check_step(self):
         """Custom sanity check paths for icc."""
 
-        binprefix = "bin/intel64"
-        libprefix = "lib/intel64"
-        if LooseVersion(self.version) >= LooseVersion("2011"):
-            if LooseVersion(self.version) <= LooseVersion("2011.3.174"):
-                binprefix = "bin"
-            elif LooseVersion(self.version) >= LooseVersion("2013_sp1"):
-                binprefix = "bin"
+        binprefix = 'bin/intel64'
+        libprefix = 'lib/intel64'
+        if LooseVersion(self.version) >= LooseVersion('2011'):
+            if LooseVersion(self.version) <= LooseVersion('2011.3.174'):
+                binprefix = 'bin'
+            elif LooseVersion(self.version) >= LooseVersion('2013_sp1'):
+                binprefix = 'bin'
             else:
-                libprefix = "compiler/lib/intel64"
+                libprefix = 'compiler/lib/intel64'
 
-        binfiles = ["icc", "icpc"]
-        if LooseVersion(self.version) < LooseVersion("2014"):
-            binfiles += ["idb"]
+        binfiles = ['icc', 'icpc']
+        if LooseVersion(self.version) < LooseVersion('2014'):
+            binfiles += ['idb']
+
+        binaries = [os.path.join(binprefix, f) for f in binfiles]
+        libraries = [os.path.join(libprefix, 'lib%s' % l) for l in ['iomp5.a', 'iomp5.so']]
+        sanity_check_files = binaries + libraries
+        if LooseVersion(self.version) > LooseVersion('2015'):
+            sanity_check_files.append('include/omp.h')
 
         custom_paths = {
-            'files': [os.path.join(binprefix, x) for x in binfiles] +
-            [os.path.join(libprefix, 'lib%s' % x) for x in ['iomp5.a', 'iomp5.so']] +
-            ['include/omp.h'],
+            'files': sanity_check_files,
             'dirs': [],
         }
 
         super(EB_icc, self).sanity_check_step(custom_paths=custom_paths)
 
     def make_module_req_guess(self):
-        """Customize paths to check and add in environment.
         """
-        self.debuggerpath = None
+        Additional paths to consider for prepend-paths statements in module file
+        """
         prefix = None
-        tbbgccversion = get_tbb_gccprefix()
+
+        # guesses per environment variables
+        # some of these paths only apply to certain versions, but that doesn't really matter
+        # existence of paths is checked by module generator before 'prepend-paths' statements are included
+        guesses = {
+            'CLASSPATH': ['daal/lib/daal.jar'],
+            'CPATH': ['daal/include', 'include', 'ipp/include', 'mkl/include', 'tbb/include'],
```

Comment:
It shouldn't break anything. But I still believe `include` should be first and take preference before the others.